### ISSUE TITLE
Emails not deleted from Mail list after being downloaded from server

### ISSUE THREAD

**Issue by Lennyr7** [jojobear99/PopTrayU#87]
After having checked new emails with PopTrayU, I then download them on my PC with Thunderbird.
I then return again to PopTrayU and push Check button for new emails. Theoretically, no new emails should show up as they were all downloaded.
But, I see **half of them** (say 4 out of 8).
I repeat the Checking and this time I see again **half**, i.e. 2 out of previous 4. Repeating this again and again will show 1 of previous 2 and only then - nothing. I therefore had to Check 4 times instead of one time only.
The same process of **deleting only half of previously shown items on the email list** also happens when PopTrayU periodically checks itself for new emails, i.e. only after 4 periodic checks will the list of 8 emails disappear.
It's bothering since in the meantime the Tray Icon keeps showing the number of "waitng" emails in each account, while in reality they were all already downloaded, thus giving a false indication.

**Comment by cephira** [comment]
For anyone else who runs into this problem, try unchecking "Advanced Options->Enable Quick Checking". This seems to be working for me. There's also a fix posted in issue #36 but that doesn't appear to have been implemented.

This appears to also be a duplicate of issue #73.

**Comment by Lennyr7** [comment]
Thanks a lot for your answer!
I do have the "Quick Checking" option ON, but the problem persists.
Actually, it deletes only the half of the downloaded emails. I have to
"Check" again to get the half of the remaining emails get deleted and so
on, until only one remains and then do one more "Check" to delete it again.
Of course I can wait until the Autocheck does the same (deleting the
downloaded emails by half on each round...)
It's not a big problem, but it's annoying.
Thanks again.

**Comment by cephira** [comment]
You need to uncheck the option, meaning turn quick checking off.

**Comment by Lennyr7** [comment]
Oops, I got it wrong then.
I checked and it does work now!
Thanks a lot  !!!
